<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Naveen" failures="0" tests="3" name="blazedemo.blazedemo.v1.testcases.FunctionalTests" time="3.541" errors="1" timestamp="2021-08-11T20:35:11 IST" skipped="2">
  <testcase classname="blazedemo.blazedemo.v1.testcases.FunctionalTests" name="checkAllAvailableAirlinesDisplayedonReservePage" time="0.000">
    <skipped/>
  </testcase> <!-- checkAllAvailableAirlinesDisplayedonReservePage -->
  <system-out/>
  <testcase classname="blazedemo.blazedemo.v1.testcases.FunctionalTests" name="setUp" time="3.541">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)&quot; because &quot;this.logger&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because "this.logger" is null
at blazedemo.blazedemo.v1.testcases.BaseClass.setUp(BaseClass.java:62)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </error>
  </testcase> <!-- setUp -->
  <system-out/>
  <testcase classname="blazedemo.blazedemo.v1.testcases.FunctionalTests" name="checkThePriceBeforeAndAfterFlighSelection" time="0.000">
    <skipped/>
  </testcase> <!-- checkThePriceBeforeAndAfterFlighSelection -->
  <system-out/>
</testsuite> <!-- blazedemo.blazedemo.v1.testcases.FunctionalTests -->
